tests/repository_data/uptane_director_root.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
That was a change made after the fork occurred, and will be another thing to unite in the merge.